src/commonMain/kotlin/exchange.dydx.abacus/processor/markets/OrderbookProcessor.kt

```diff
@@ -510,43 +517,55 @@ internal class OrderbookProcessor(parser: ParserProtocol) : BaseProcessor(parser
     }
 
     fun group(orderbook: List<Any>?, grouping: Double): List<Any>? {
-        return if (orderbook != null && orderbook.size > 0) {
+        return if (!orderbook.isNullOrEmpty()) {
```

Comment:
[discussion/suggestion] IMO this is one those functions that has enough math and logic where it makes sense to test separately. we should probably be actively testing edge cases and difficult to reason-about situations like stuff that's been commented here.

i tried to start a pattern with adding [the processor tests](https://github.com/dydxprotocol/v4-abacus/tree/main/src/commonTest/kotlin/exchange.dydx.abacus/processor) to the commonTest folder.

Reply:
I think the existing orderbook tests hit the main points pretty well............and I feel like it would take me forever to set up new tests.

Reply:
Yeah the orderbook tests basically hit this directly